artifacts/deployment/etcdproxy-controller.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-apiserver-storage
+spec:
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: etcdproxy-clusterrole
+rules:
+- apiGroups: ["extensions", "apps"]
+  resources: ["deployments"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["etcd.xmudrii.com"]
+  resources: ["etcdstorages"]
+  verbs: ["get", "watch", "list"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: etcdproxy-crb
+subjects:
+- kind: ServiceAccount
+  name: default # Name is case sensitive
```

Comment:
don't use default.  choose a specific name.

Reply:
By that I mean, create a service account just for running this controller and grant it specific access.

Reply:
Added manifest for creating ServiceAccount and modified other manifests to use it.